docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -0,0 +1,301 @@
+include::variables.adoc[]
+
+[[setup-virtualization-environment]]
+= Setting Up the Virtualization Environment
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:next-steps: Proceed to xref:../getting-started/installing.adoc#[Installing {project}] once your hypervisor has been installed and configured.
+
+toc::[]
+
+[[setup-virtualization-environment-overview]]
+== Overview
+
+Follow the appropriate procedure to set up virtualization for your particular operating system and hypervisor.
+{project} uses Docker Machine and its driver plug-in architecture to provide a consistent way to manage the OpenShift VM.
+
+Some hypervisors require manual installation of the driver plug-in.
+{project} embeds VirtualBox drivers, so no additional steps are required to configure them.
+However, VirtualBox will need to be identified to {project} via the `--vm-driver virtualbox` flag or persistant configuration settings.
+See xref:setting-up-virtualbox-driver[Setting Up {project} to Use VirtualBox] for more information.
+However, the `--vm-driver virtualbox` flag will be required later to identify VirtualBox to {project}.
```

Comment:
This sentence is confusing. What do you want to communicate here?

Reply:
This sentence was inadvertently left in, it should be removed entirely. :)